onnx_coreml/_shapeinference.py

```diff
@@ -10,9 +10,9 @@
 def infer_shapes_and_types(graph):  # type: (GraphProto) -> None
     model = helper.make_model(graph)
     c2_backend = _load_model(model)
-    with c2_backend.workspace:
-        inference_result = _infer(c2_backend)
-        _add_shapes_and_types_to_graph(graph, inference_result)
+    #TODO Gate the following two lines with "with c2_backend.workspace:" once fixed
```

Comment:
If you refer to `c2_backend.workspace` directly, you don't need `with`. And can you share with me the onnx model that causes the issue? I was not able to reproduce with shufflenet model.

Reply:
yeah in fact c2_backend.workspace is never intended to be used as a context manager

Reply:
I used the candy model from pytorch/examples